rfcs/models/identity/decentralized-network-identity-discovery-management.md

```diff
@@ -0,0 +1,179 @@
+
+What we cannot know beforehand is what configuration of DID registries or VC issuers will exist at any given time nor should we make any assumptions that will limit our choices in the future. We can also not know the complete trust calculus beforehand that tells us how to connect two interoperating networks via common DID registries and VC issuers. We can only rely on the existence of DID and VC standards that everyone will eventually converges to. But for the purpose of our design, we can (and will) select a particular configuration that shows how the basis of interoperation can be built using DIDs and VCs. Our mechanisms can be easily adapted for different configurations of DID registries and VC issuers than the one we will describe in detail in this RFC.
+
+
+# Distributed Identity Management Architecture
+
+The below figure (Fig. 3) illustrates our identity plane architecture:
+* A set of identity providers and verifiers to establish a trust basis for exchange of identity and any other network configuration information. We assume, without loss of generality, that such identity providers belong to networks which we term as _Interoperation Identity Networks (IINs)_.
+* A set of agents that act on behalf of members (participant units) within a network that collectively supply and fetch membership, identity, and any other configuration information, of the counterparty network.
+
+<div align="center">
+<img src="../../resources/images/identity-plane.jpg" width=70%>
+<br/>
+<figcaption>Fig.3 - Identity plane architecture overview</figcaption><br/>
+</div>
+<br/>
+
+_Some Notes on IINs (Pertaining to the Design Choices Described Earlier)_:
+* _The term 'Interoperation Identity Network' is a placeholder to describe a unit of our proposed architecture, and is subject to change in the future. A term that may replace it is 'Distributed Identity Registry', which captures both the role and the nature of this entity, and which practitioners may already be familiar with_.
+* _An IIN is not a new concept or system we are inventing for interoperation purposes but is rather an extrapolation of the concept of a DID registry, many varieties of which exist today, to a distributed system that may maintain DID records through consensus among a committee on a shared ledger rather than maintain them in a centralized repository_.

```

Comment:
@ermyas I can't locate your comment about the terminology (using "IIN" for a DID Registry"), but I just wanted to mention that we had considered changing the name earlier this year to DIR (Distributed Identity Registry) as a shorthand. The reason to use a special acronym I suppose is simply that we are considering adding some capabilities (like DID validation checks using contracts) to these registries, which are functions DID Registries typically don't perform. So we'd be recommending a new abstraction (which would be an augmentation of the DID Registry) for interoperation purposes, while still supporting legacy DID Registries.

IIN just happened to be the term we've been using for a while, in diagrams and conference papers, so we just decided to continue using it. We can change the name to DIR, but then we'd have to rename everything throughout these docs.

Reply:
i see. ok

Reply:
We recognize the need to reassure users that we are leveraging the existing identity infrastructure - DID Registries. We have hinted this on the [main page](https://github.com/ghoshbishakh/weaver-dlt-interoperability/blob/rfc-identity/rfcs/models/identity/decentralized-network-identity-discovery-management.md#distributed-identity-management-architecture). We expand on this and provide more comprehensive illustration in the subsequent PR.